syft/frameworks/torch/tensors/Polynomial.py

```diff
@@ -0,0 +1,77 @@
+class PolynomialTensor:
+
+    """Tensor type which provides polynomial approximation functionalities using Taylor Series expansion
+       since computing exact functions could impose a overhead on computation
+    """
+
+    def sigmoid(self, x: float) -> float:
+
+        """Parameters:
+            
+           x: A float value
+           
+           return: 
+               
+           approximation of the sigmoid function as a float"""
+
+        return (1 / 2) + (x / 4) - (x ** 3 / 48) + (x ** 5 / 480)
+
+    def exp(self, x: float, order: int) -> float:
```

Comment:
same as above, this should either operate on self or be a class method. Additionaly should other functions have the option to choose order or should we pick a standard order for the entire class?